index.md

```diff
@@ -3,15 +3,55 @@ layout: lesson
 root: .  # Is the only page that doesn't follow the pattern /:path/index.html
 permalink: index.html  # Is the only page that doesn't follow the pattern /:path/index.html
 ---
-FIXME: home page introduction
 
-<!-- this is an html comment -->
+This tutorial is built to teach you ESMValTool.  
 
-{% comment %} This is a comment in Liquid {% endcomment %}
+> ## What will you learn in this course
+>
+>   - What is ESMValTool
+>
+>   - How to install ESMValTool
+>
+>   - How to run ESMValTool
+>
+>   - How to develop your own diagnostics and recipes
+>
+>   - How to contribute your recipes and diagnostics back into ESMValTool
+>
+{: .checklist}
 
 > ## Prerequisites
 >
-> FIXME
+> Basic understanding of git (optional - but useful)
+> Basic understanding of your preferred command line interface (ie a bash terminal)
+> Access to CMIP data
+> Access to a suitable computing system (eg Jasmin)
+> Github account (optional, but useful!)
 {: .prereq}
 
+> ## Main things you need to know before starting this course
+>
+> 1. This tutorial can be taken online independently or taught by one of our instructors.
+> 2. Please check the common issues page if you get stuck [Link to common issues page]. Otherwise, help is always available from ESMValTool developers via the github issues page.  
+> 3. This tutorial includes several advanced lessons after the conclusion. These advanced lessons should be treated like “mini-tutorials”, and include aspects like “developing your own diagnostic” or “how to include observations”.
+> 4. Don’t be alarmed if you can’t work through the entire tutorial in one sitting. It may take some time to get used to working with ESMValTool.
+{: .checklist}
```

Comment:
I would not put this in a box but just in the main content: 
```suggestion

### Main things you need to know before starting this course

1. This tutorial can be taken online independently or taught by one of our instructors.
2. Please check the common issues page if you get stuck [Link to common issues page]. Otherwise, help is always available from ESMValTool developers via the github issues page.  
3. This tutorial includes several advanced lessons after the conclusion. These advanced lessons should be treated like “mini-tutorials”, and include aspects like “developing your own diagnostic” or “how to include observations”.
4. Don’t be alarmed if you can’t work through the entire tutorial in one sitting. It may take some time to get used to working with ESMValTool.

```